### ISSUE TITLE
Unnecessary event args due to unicorn/prevent-abbreviations

### ISSUE THREAD

**Issue by kkmuffme** [sindresorhus/eslint-plugin-unicorn#1430]
More a side-effect than a bug of unicorn/prevent-abbreviations.
Due to this rule, lots of devs just changed the e in an eventlistener to the (global) event variable.

This rule or a separate rule should check to avoid unnecessary passing of the global event variable in an eventlistener.

unicorn/prevent-abbreviations

```js
document.body.addEventListener( 'mousedown', ( event ) => {
    if ( event.target.indexOf( 'giraffe' ) ) {
        alert( 'It's a giraffe!' );
    }
}, { passive: true });
```

The `( event )` is unnecessary, since event is a globally available variable with the event details, so just `()` would suffice here.

**Comment by fisker** [comment]
The idea is  good, but we can forbid with this https://www.npmjs.com/package/confusing-browser-globals

**Comment by sindresorhus** [comment]
I disagree. Even if there's a global with the same name, it's clearer to use the shadowing local variable.

**Comment by fisker** [comment]
> This rule or a separate rule should check to avoid unnecessary passing of the global event variable in an eventlistener.

@sindresorhus I think he mean to forbid global `event`.

**Comment by kkmuffme** [comment]
Ok, then at least should suggest to use `event` instead of `event_` ?

**Comment by sindresorhus** [comment]
Yes. We should do that.

**Comment by fisker** [comment]
We already did, 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2d052526d2f8e926487a957bb27e8a88e95b96a5/test/prevent-abbreviations.mjs#L1809-L1831

You must use `event` somewhere else.
